vtex/faststore
# Chore: Refactors `Gift` Content component and replaces on Core

## What's the purpose of this pull request?

- Refactors `GiftContent` component to receive the following props:  `productName`, `price`, `badgeLabel`, `titleMessage`.

## How it works?

Everything should look as before. (Visually)

## How to test it?

1. Test the component and its documentation:
run yarn start in app/docs folder and checks the `Gift` component or test it using the doc preview link:
[Gift Component](https://faststore-site-git-chore-refactor-gift-compone-f01e25-faststore.vercel.app/components/molecules/gift)

2. Test the component replacement in the application:
run yarn dev in packages/core -> Adds 3 items to the cart -> the `Gift` component should appear in the cart

<img width="300" alt="image" src="https://user-images.githubusercontent.com/3356699/219411822-d91318f4-2f30-4b4d-bdda-252976fbe49b.png">
